dm/config/subtask.go

```diff
@@ -116,6 +116,7 @@ type SubTaskConfig struct {
 	HeartbeatUpdateInterval int    `toml:"heartbeat-update-interval" json:"heartbeat-update-interval"`
 	HeartbeatReportInterval int    `toml:"heartbeat-report-interval" json:"heartbeat-report-interval"`
 	EnableHeartbeat         bool   `toml:"enable-heartbeat" json:"enable-heartbeat"`
+	ResumeResetConn         bool   `toml:"resume-reset-conn" json:"resume-reset-conn"`
```

Comment:
I think we can remove this config, and reset connection as default.

Reply:
agree